packages/form-core/src/FieldApi.ts

```diff
@@ -410,3 +427,11 @@ function normalizeError(rawError?: ValidationError) {
 
   return undefined
 }
+
+function getErrorMapKey(cause: ValidationCause) {
+  return cause === 'submit'
```

Comment:
I'd rather this be a `switch`/`case` as it reads a lot nicer than nested turneries.

Reply:
Let's also add `onMount`

Reply:
Good call, switch/case sounds much nicer